### DISCUSSION TITLE
Prowler Issue

### DISCUSSION THREAD

**Discussion by jatinwadhwa27** [prowler-cloud/prowler#2564] [General]
2023-07-10 12:14:12,691 [File: utils.py:15] 	[Module: utils]	 CRITICAL: xxxxxxxxxxxxx.html: OSError[13]

When I run the prowler tool I get this error for my aws tenant

**Comment by toniblyx** [comment]
Hi @jatinwadhwa27 that is due to the limit of file descriptors allowed in that session in your OS, look at here to fix it https://docs.prowler.cloud/en/latest/troubleshooting/

**Comment by jatinwadhwa27** [comment]
Hi Toni
Thanks for this but I running prowler on ubuntu 20.0.4

**Comment by toniblyx** [comment]
That issue apply to any linux/unix OS.

**Comment by jatinwadhwa27** [comment]
Hello Toni

I am still getting the same error after using ulimit -n 1000.
My ubuntu machine is 1 core, 8 GB memory.

Best regards
Jatin

On Mon, Jul 10, 2023 at 6:57 PM Toni de la Fuente ***@***.***>
wrote:

> That issue apply to any linux/unix OS.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/prowler-cloud/prowler/discussions/2564#discussioncomment-6404522>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBFQLGJQ4KODHWFQIBH5ZT3XPP7MZANCNFSM6AAAAAA2EPCURI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Jatin Wadhwa
Infosec Consultant
Indisec IS Consultants Pvt
+91 9818231221

**Comment by toniblyx** [comment]
can you add `--log-level ERROR` to see what else is going on?

**Comment by jatinwadhwa27** [comment]
I have used the log level error but didnt find any specific issues in the json collected.

**Comment by jatinwadhwa27** [comment]
Hi Toni

I have sorted the issue out. I upgraded the prowler to 3.7.0 the latest one
and also changed the ulimit to 4096 from 1000 as before it worked like a
charm
Thank you
Any article if you cant point me toward which can help build a dashboard
for my prowler reports

On Mon, Jul 10, 2023 at 9:05 PM Toni de la Fuente ***@***.***>
wrote:

> can you add --log-level ERROR to see what else is going on?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/prowler-cloud/prowler/discussions/2564#discussioncomment-6406095>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBFQLGPFATHMZTAXKBFGYJ3XPQOMXANCNFSM6AAAAAA2EPCURI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Jatin Wadhwa
Infosec Consultant
Indisec IS Consultants Pvt
+91 9818231221

**Reply by toniblyx** [reply]
Glad to hear that you managed to fix it. Thanks for let us know.

For dashboards you can see if the html default report is enough but sometimes more is needed. Some helpful resources on how to generate dashboards, see here https://docs.aws.amazon.com/prescriptive-guidance/latest/patterns/create-a-consolidated-report-of-prowler-security-findings-from-multiple-aws-accounts.html#create-a-consolidated-report-of-prowler-security-findings-from-multiple-aws-accounts-additional. The excel template there can be downloaded from here https://github.com/aws-samples/multi-account-security-assessment-via-prowler. There is also a workshop you can take as a reference for more dashboards with QuickSight here https://catalog.us-east-1.prod.workshops.aws/workshops/b1cdc52b-eb11-44ed-8dc8-9dfe5fb254f5/en-US
